### ISSUE TITLE
Change remappings for @chainlink

### ISSUE THREAD

**Issue by andrejrakic** [smartcontractkit/foundry-starter-kit#7]
Should we modify remappings for `@chainlink/contracts` to point to the official [chainlink](https://github.com/smartcontractkit/chainlink) repo instead of [chainlink-brownie-contracts](https://github.com/smartcontractkit/chainlink-brownie-contracts) since the first one is more actively maintained? The change will include removing chainlink-brownie-contracts and forge installing `smartcontractkit/chainlink` which will increase the size of the `lib` folder, which is the possible downside of this idea. 

**TLDR:**
Update `@chainlink/=lib/chainlink-brownie-contracts/` to `@chainlink/=lib/chainlink`?

**Comment by PatrickAlphaC** [comment]
The `lib/chainlink-brownie-contracts/` is a LOT smaller than the full repo and makes it download a lot quicker. It's also just a subset of the official contracts from NPM anyhow.

If others think it makes sense to switch I'm down, but the extra speed from the smaller repo is... _really_ nice.
